t/print_errors.t

```diff
@@ -0,0 +1,76 @@
+use strict;
+use warnings;
+use lib 't/lib';
+use PkgConfig::Capture;
+use Test::More;
+use PkgConfig;
+
+note "DEFAULT_SEARCH_PATH = $_" for @PkgConfig::DEFAULT_SEARCH_PATH;
+
+# adjust to test with real pkg-config
+my @pkg_config = ( $^X, $INC{'PkgConfig.pm'} );
+#my @pkg_config = ( 'pkg-config' );
+
+
+my $nonexistent_lib = 'rubbish-no-exists';
+
+subtest 'ppkg-config --print-errors with non-existent lib' => sub {
+
+  my @command = ( @pkg_config, '--print-errors', $nonexistent_lib );
+
+  note "% @command";
+  my($out, $err, $ret) = capture {
+    system @command;
+    $?;
+  };
+
+  is $ret, 256;
+  is $out, "";
+  like $err,
+    qr/^Can't find $nonexistent_lib.pc in any of /,
+    "errors went to stderr";
+  note "out: $out" if defined $out;
+  note "err: $err" if defined $err;
+
```

Comment:
In older versions of `Test::More` you need to add either a plan or done_testing (see error here: https://travis-ci.org/github/PerlPkgConfig/perl-PkgConfig/jobs/742588971#L308 ).  Normally I require a more recent version of Test::More but for PkgConfig.pm one of the project goals is to avoid non-core deps.

Reply:
I prefer `done_testing` in this case fwiw

Reply:
Added done_testing at the end of each subtest.